ppgan/engine/trainer.py

```diff
@@ -21,6 +21,7 @@
 
 import paddle
 from paddle.distributed import ParallelEnv
+import visualdl
```

Comment:
Put this line before line 53. so that the code can run  run successfully, if not install VisualDL and set enable_visualdl False.